gui/slick/interfaces/default/IRC.mako

```diff
@@ -1,15 +1,17 @@
-#import sickbeard
-#from sickbeard.common import *
+<%
+    import sickbeard
+    from sickbeard.common import *
 
-#import os.path
-#include $os.path.join($sickbeard.PROG_DIR, "gui/slick/interfaces/default/inc_top.tmpl")
+    import os.path
+    include $os.path.join($sickbeard.PROG_DIR, "gui/slick/interfaces/default/inc_top.tmpl")
```

Comment:
So they can be removed here too :)
I was going over all the changes to see if every thing was updated.